eZ/Publish/API/Repository/Values/User/User.php

```diff
@@ -24,6 +24,9 @@
  */
 abstract class User extends Content implements UserReference
 {
+    /**
+     * @var int[] List of supported (by default) hash types.
+     */
```

Comment:
Side note: to speed up checks if hash is supported this can be turned into `array<int, int>`. Then to check support for hash you'd just see if:
```php
isset(User::SUPPORTED_PASSWORD_HASHES[$hashType]);
```

Reply:
> Premature optimization is the root of all evil

D. Knuth